.github/workflows/docker-build-push.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta-installer.outputs.tags }}
           labels: ${{ steps.meta-installer.outputs.labels }}
-  gcp-manager:
+  csi-gcp:
```

Comment:
```suggestion
  sci-gcp:
```

Reply:
I did a search and I used csi a lot, I pushed over your commit and will tackle this by doing find and going through all occurences